### PR TITLE
Fix the major issues outlined by SonarQube and remove duplication

### PR DESCRIPTION
**Pull request by iainduncani**
SonarQube identified 3 bugs such as null checks and 3 vunerabilities so
these have been removed as well as some minor and quick to fix smells in
those files (such as use of Sysout).

Also remove the duplication that was added in RunnableJarTest

---
This change is [Reviewable](https://reviewable.io/reviews/wasdev/tool.accelerate.core/51)

### CODE REVIEW

File: liberty-starter-application/src/main/java/com/ibm/liberty/starter/PomModifier.java [hunk 1]
```diff
@@ -227,4 +238,14 @@ private void appendRepoUrl() throws IOException {
         }
     }
 
+    private static class UnableToFindNodeExcpetion extends Exception {
```
Comment (katheris) [inline liberty-starter-application/src/main/java/com/ibm/liberty/starter/PomModifier.java]: Mis-spell

**Comment by katheris** [comment]
Reviewed 8 of 9 files at r1.
Review status: 8 of 9 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/wasdev/tool.accelerate.core/51)*

**Comment by katheris** [comment]
Reviewed 1 of 9 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/wasdev/tool.accelerate.core/51)*

**Comment by iainduncani** [comment]
*[liberty-starter-application/src/main/java/com/ibm/liberty/starter/PomModifier.java, line 241 \[r1\]](https://reviewable.io:443/reviews/wasdev/tool.accelerate.core/51#-KJkE5UbdYeVgSRVtBmR-r1-241:-KJv1zRBsQR0SZ-wQ3PD:-499146009) ([raw file](https://github.com/wasdev/tool.accelerate.core/blob/8091ac675634f64c2832a62280ad3ab9fa802e45/liberty-starter-application/src/main/java/com/ibm/liberty/starter/PomModifier.java#L241)):*
<details><summary><i>Previously, katheris (Katherine Stanley) wrote…</i></summary>
> Mis-spell
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/wasdev/tool.accelerate.core/51)*

**Comment by katheris** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/wasdev/tool.accelerate.core/51)*

**Pull request merged**